src/lib.rs

```diff
@@ -140,9 +169,16 @@ impl HexView {
             cursor: Vec2::zero(),
             data: data.into_iter().map(|u| *u.borrow()).collect(),
             state: DisplayState::Disabled,
+            config: HexViewConfig::default()
         }
     }
 
+    ///Change configuration values for spacers and separators
+    pub fn config(mut self, config: HexViewConfig) -> Self {
+        self.config = config;
+        self
+    }
```

Comment:
I also would like to see a `set_config` function which accepts a `&mut self` so modifying while not chaining things is easier.

Reply:
Let me know if [this](https://github.com/palako/cursive_hexview/commit/1539f7e76ad27b4c684b7fe2677bb2c16ad99b08) is good.